package.json

```diff
@@ -15,7 +15,8 @@
     "lint:fix": "eslint . --fix",
     "format:fix": "prettier --write \"**/*.{ts,tsx,json,scss,css}\"",
     "format:check": "prettier --check \"**/*.{ts,tsx,json,scss,css}\"",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "schema": "graphql-markdown http://localhost:4000/graphql > Schema.md"
```

Comment:
1. This should export to a file in the `docs/` directory. We have too much stuff in root.
2. This will also need to be documented in CONTRIBUTING.md